openedx_proversity_reports/reports/last_page_accessed.py

```diff
@@ -130,18 +144,17 @@ def get_exit_count_data(last_page_data, course_list):
         except InvalidKeyError:
             continue
 
-        # Getting all students enrolled in the course except staff users
-        enrolled_students = User.objects.filter(
+        # Getting all users enrolled in the course.
```

Comment:
this only returns the first user

Reply:
I just need the first user, i don't need other users, since i need the course structure once.